tests/app/utilities/test_factory.py

```diff
@@ -32,5 +32,10 @@ def test_register_all(self):
         self.assertIsInstance(factory.create("test-a"), TestClassA)
         self.assertIsInstance(factory.create("test-b"), TestClassB)
 
+    def test_create_value_error(self):
```

Comment:
I can't find anywhere that we actually use `Factory` within the code.
Maybe it should just be removed?